spec/traits.dd

```diff
@@ -400,6 +401,36 @@ static assert(!__traits(isTemplate,foo!int()));
 static assert(!__traits(isTemplate,"string"));
 ---
 
+$(H2 $(GNAME isZeroInit))
+
+        $(P Takes one argument which must be a type. If the type's
+        $(LINK2 property.html#init, default initializer) has no non-zero bits
+        then $(D true) is returned, otherwise $(D false). This always will
+        return $(D true) for a class $(D C) because $(D C.init) is a null
+        reference.)
+
+---
```

Comment:
Bonus points for:

```
$(SPEC_RUNNABLE_EXAMPLE_COMPILE
---
...
---
)
```

because then the spec tester will ensure that this code continues to work. I know that we have unit tests, but code that doesn't get tested slowly breaks over time :/